pset-functions.js

```diff
@@ -115,6 +209,26 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+function isPalindrome(word){
+  // check word left to right
+  for(let i=0; i < word.length; i++){
+    // check word right to left
+    for(let j=word.length -1-i; j> word.length -2-i;j--){
```

Comment:
This is a very clever way of checking opposite ends of a string to see if they match! Be careful using a nested loop however, it can affect how quickly how your code will run. [Here](https://github.com/jsd20200127/hw-02-functions-solution/blob/e87c29073cf7285a5045ec8708561f279474f2d2/pset-functions.js#L163) is an alternate solution (purely for your benefit) that uses different js built-in methods.